### DISCUSSION TITLE
How to show part of the next slide

### DISCUSSION THREAD

**Discussion by SvenBudak** [nolimits4web/swiper#5283] [Q&A] [answered]
### Clear and concise description of the problem

Is this: https://github.com/nolimits4web/Swiper/issues/1801 still the only way to show a part of the next slide?
I checked the docs multiple times... but i was not able to find a simple config entry for this.
if this isnt available, is this feature planed?

### Suggested solution

add something like `offsetNextSlide: 64` to show 64px of the next slide.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by rolandszpond** [comment, marked as answer]
Use decimals: `slidesPerView={1.2}`

**Reply by SvenBudak** [reply]
Yes... sometimes the solutions are easier than you think...
